python/packages/autogen-ext/src/autogen_ext/tools/mcp/_session.py

```diff
@@ -26,3 +31,105 @@ async def create_mcp_server_session(
         async with sse_client(**server_params.model_dump()) as (read, write):
             async with ClientSession(read_stream=read, write_stream=write) as session:
                 yield session
+
+
+class McpSessionConfig(BaseModel):
+    server_params: McpServerParams
+
+
+class McpSessionActor(ComponentBase[BaseModel], Component[McpSessionConfig]):
```

Comment:
The `McpSessionActor` should be used as an async context manager.

it is very easy for user to accidently forget calling `close()`. 

I think the new usage should be like this:

```python
async McpSessionActor(server_params) as session:
  tools = mcp_server_tools(sessions)
  # rest of code.
```

Otherwise we will need to call `close()` on every tools directly, which is fine in some cases, but it is really easy to forget. So we should support the async context manager usage.